README.md

```diff
@@ -321,7 +321,9 @@ For example, adding a new data member to a struct is an ABI Breaking Change as i
 In CCCL, the guarantees about ABI are as follows:
 
 - Symbols in the `thrust::` and `cub::` namespaces may break ABI at any time without warning.
-- The ABI of `cub::` symbols includes the CUDA architectures used for compilation. Therefore, a single `cub::` symbol may have a different ABI if compiled with different architectures.
+- The ABI of `thrust::` and `cub::` symbols includes the CUDA architectures used for compilation. Therefore, a single `cub::` symbol may have a different ABI if:
+    - compiled with different architectures
+    - compiled with nvcc vs host compiler only
```

Comment:
This should be something like `compiled from a CUDA source file compared to a C++ file`. By a literal reading of this line `nvcc -x c++` would infer same ABI as `nvcc -x cu` which isn't correct. It also fails to distinguish how to reason when `clang++` is both the C++ and CUDA compiler ( no nvcc at all ).